create_manifest.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+"""
+/*******************************************************************************
+*   Ledger SDK
+*   (c) 2022 Ledger
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+/*******************************************************************************
+"""
+
+import sys
+import argparse
+import tomli_w
+
+def auto_int(x):
+	return int(x, 0)
+
+def get_argparser():
+	parser = argparse.ArgumentParser(description="Load an app onto the device from a hex file.")
```

Comment:
Can you add a comment explaining this is taken as is from loadApp.py parser which can expect quite a lot of params...

Reply:
Maybe you could even import `get_argparser()` from `ledgerblue.loadApp`? It would make things more obvious. What do you think?

Reply:
So it means that users will have to use ledgerblue but the point is to use ledgerctl this is why I did not want to add ledgerblue dependancies. But I can add a comment and copy/paste the arguments in the same order that there are in ledgerblue if you want